src/misc/unsafeEntries.ts

```diff
@@ -0,0 +1,3 @@
+export const unsafeEntries = Object.entries as <T>(
+  obj: T,
+) => [keyof T, T[keyof T]][];
```

Comment:
Seems unused

Reply:
Nice catch. I moved it out to a separate file, and few commits later removed the usage. Removing it for now then.